pydax/loaders/_text.py

```diff
@@ -34,9 +33,8 @@ def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDic
         :raises TypeError: ``path`` is not a path object.
         """
 
-        if not isinstance(path, (str, os.PathLike)):
-            # In Python 3.8, this can be done with isinstance(path, typing.get_args(_typing.PathLike))
-            raise TypeError(f'Unsupported path type "{type(path)}".')
+        self.check_path(path)
 
         encoding = options.get('encoding', 'utf-8')
-        return pathlib.Path(path).read_text(encoding=encoding)
+        # We can remove the error suppression once Dict[str, str] handling is added
+        return pathlib.Path(path).read_text(encoding=encoding)  # type: ignore [arg-type]
```

Comment:
You can use `typing.cast` to avoid ignoring the error

Reply:
got it thanks!